### DISCUSSION TITLE
Wildcard cert and subdomains

### DISCUSSION THREAD

**Discussion by SergeiLitvinov** [nginx-proxy/acme-companion#869] [Q&A] [answered]
Hi!

Would subdomains `*.foo.bar` get cert if I have wildcard сert for `foo.bar`?

**Comment by buchdag** [comment, marked as answer]
If you have an existing certificate, yes you can use it with nginx-proxy by [following the corresponding instructions](https://github.com/nginx-proxy/nginx-proxy#wildcard-certificates).

Please note that if you use existing certificate(s), this container (nginx-proxy/acme-companion) isn't involved at all.
